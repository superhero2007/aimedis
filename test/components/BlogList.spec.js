import React from 'react';
import { mount } from 'enzyme';

import BlogList from 'components/BlogList';

function setup() {
  return mount(<BlogList />);
}

describe('BlogList', () => {
  const wrapper = setup();

  it('should be a Component', () => {
    expect(wrapper.instance() instanceof React.Component).toBe(true);
  });

  it('should render properly', () => {
    expect(wrapper.find('.app__blog__list')).toExist();
    expect(wrapper.find('.app__blog__list__content')).toExist();
    expect(wrapper.find('.app__blog__list__pagination')).toExist();
  });
});
