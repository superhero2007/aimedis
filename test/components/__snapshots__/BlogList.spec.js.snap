// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogList Calls minium render properly 1`] = `"<div class=\\"app__blog__list\\"><div class=\\"app__blog__list__content\\"><div class=\\"row\\"><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/1\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></div></a></div></div></div><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__blog__list__pagination d-none d-lg-flex\\"></div></div></div>"`;

exports[`BlogList Calls small render properly 1`] = `"<div class=\\"app__blog__list\\"><div class=\\"app__blog__list__content\\"><div class=\\"row\\"><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/1\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/2\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/3\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/4\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/5\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/6\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/7\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/8\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/9\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></div></a></div></div></div><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__blog__list__pagination d-none d-lg-flex\\"><div class=\\"app__blog__list__pagination__prev\\">&lt;</div><div class=\\"app__blog__list__pagination__item active\\">1</div><div class=\\"app__blog__list__pagination__item\\">2</div><div class=\\"app__blog__list__pagination__item\\">3</div><div class=\\"app__blog__list__pagination__next\\">&gt;</div></div></div></div>"`;

exports[`BlogList should render properly 1`] = `"<div class=\\"app__blog__list\\"><div class=\\"app__blog__list__content\\"><div class=\\"row\\"><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/1\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/2\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/3\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/4\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/5\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/6\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/7\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/8\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/9\\"><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></div></a></div></div></div><div class=\\"animated\\" style=\\"opacity: 0;\\"><div class=\\"app__blog__list__pagination d-none d-lg-flex\\"><div class=\\"app__blog__list__pagination__prev\\">&lt;</div><div class=\\"app__blog__list__pagination__item active\\">1</div><div class=\\"app__blog__list__pagination__item\\">2</div><div class=\\"app__blog__list__pagination__item\\">3</div><div class=\\"app__blog__list__pagination__item\\">4</div><div class=\\"app__blog__list__pagination__item\\">5</div><div class=\\"app__blog__list__pagination__abbr abbr\\">...</div><div class=\\"app__blog__list__pagination__item\\">12</div><div class=\\"app__blog__list__pagination__next\\">&gt;</div></div></div></div>"`;
